ui/components/multichain/account-list-item/index.scss

```diff
@@ -0,0 +1,32 @@
+.multichain-account-list-item {
+  position: relative;
+  width: 100%;
+
+  &:not(.account-list-item--selected) {
+    &:hover,
+    &:focus-within {
+      background: var(--color-background-default-hover);
+    }
+  }
+
+  &__selected-indicator {
+    width: 4px;
+    height: calc(100% - 8px);
+    position: absolute;
+    top: 4px;
+    left: 4px;
+  }
+
+  &__content {
+    overflow: hidden;
+    flex: 1;
+  }
+
+  .currency-display-component {
+    flex-wrap: nowrap;
+  }
+
+  &__tooltip {
+    display: inline;
+  }
```

Comment:
I think we can remove the css for tooltip

Reply:
This is required for the `chaos` item, otherwise the ellipsis doesn't happen.